README.md

```diff
@@ -22,15 +37,16 @@ git clone https://github.com/purescript-halogen/purescript-halogen-template.git
 cd halogen-project
 ```
 
-Then, install the PureScript compiler, the [Spago](https://github.com/purescript/spago) package manager and build tool, and [Webpack](https://github.com/webpack/webpack) bundler locally:
+Then, install the PureScript compiler, the [Spago](https://github.com/purescript/spago) package manager and build tool, and the [Parcel](https://github.com/parcel-bundler/parcel) bundler globally:
 
-```shell
-npm install
 ```
+npm install -g purescript spago parcel
```

Comment:
Perhaps we should flip this around and prefer the local toolchain to the global one -- introduce how to do the local toolchain, and afterwards add "if you just want to install tools globally, ...". I'd consider it best practice to keep tooling local where possible.

Reply:
I go back and forward on this.

The user is _guaranteed_ to encounter the negatives (more disk space and time) of local installation. But only _risks_ encountering the negatives (version mismatch) of global installation.
Some space and time numbers: https://github.com/purescript-contrib/purescript-book/issues/130#issuecomment-625612577

Reply:
Maybe we can just rephrase this so as not to privilege one over the other, saying something like:

> You can install PureScript tooling globally, so it works across projects, or locally, so that each project uses specific versions of the tools. Regardless of which approach you choose, this template requires the PureScript compiler, the Spago package manager and build tool, and the Parcel bundler.
> 
> To install the toolchain globally:
> 
> To install the toolchain locally: